okgo/check/check_test.go

```diff
@@ -32,6 +33,9 @@ type inMemoryChecker struct {
 }
 
 func (i *inMemoryChecker) Type() (okgo.CheckerType, error) {
+	if i.checkerType == "error_on_type_check" {
+		return "", errors.New("hi")
```

Comment:
I know this is just a test, but maybe update the error message to be a bit more descriptive ("test error on Type()" or something?)

Reply:
Whoops, sorry fixed